### ISSUE TITLE
Failed to build: "Read must defined only once in the type namespace of this module."

### ISSUE THREAD

**Issue by KazeEnji** [Tylous/Freeze.rs#3]
Hello,

Attempting to build with a Havoc C2 implant and I'm receiving this error:

```
./Freeze-rs -s -E ELZMA -I /home/kali/Documents/FreezProj/https.bin -O /home/kali/Documents/FreezeProj/frozen.exe

    ___________                                                      
    \_   _____/______   ____   ____ ________ ____     _______  ______
     |    __) \_  __ \_/ __ \_/ __ \\___   // __ \    \_  __ \/  ___/
     |     \   |  | \/\  ___/\  ___/ /    /\  ___/     |  | \/\___ \ 
     \___  /   |__|    \___  >\___  >_____ \\___  > /\ |__|  /____  >
         \/                \/     \/      \/    \/  \/            \/    
                                        (@Tyl0us)
    Soon they will learn that revenge is a dish... best served COLD & Rusty...
    
    
[!] Selected Process to Suspend: notepad.exe
[*] Encrypting Shellcode Using ELZMA Encryption
[*] Shellcode Encrypted
[*] Created new Rust project: /home/kali/Documents/FreezeProj/frozen
[*] Added dependency to Cargo.toml
error[E0252]: the name `Read` is defined multiple times
  --> src/main.rs:33:5
   |
28 | use std::io::{BufReader, Read};
   |                          ---- previous import of the trait `Read` here
...
33 | use std::io::Read;
   |     ^^^^^^^^^^^^^ `Read` reimported here
   |
   = note: `Read` must be defined only once in the type namespace of this module

For more information about this error, try `rustc --explain E0252`.
error: could not compile `frozen` (bin "frozen") due to 1 previous error
Error: 'cargo build' failed
```

**Comment by Tylous** [comment]
I see what the issue is, ill fix it shortly.

**Comment by Tylous** [comment]
pushed an update to address this.

**Comment by KazeEnji** [comment]
Thank you for updating it. New Issue popped up. Would you like me to create a new issue request or put it in this thread?

**Comment by Tylous** [comment]
doesn't matter to me

**Comment by KazeEnji** [comment]
Awesome! I'll post a new Issue so this one gets resolved.
